File: java/src/main/java/com/izerui/vo/FileTree.java
package com.izerui.vo;

import java.util.List;

public class FileTree {
	private String foldername;//文件夹名字
	private String folderpath;//文件夹实际路径
	private List<FileTree> children;//子文件夹
	public String getFoldername() {
		return foldername;
	}
	public void setFoldername(String foldername) {
		this.foldername = foldername;
	}
	public List<FileTree> getChildren() {
		return children;
	}
	public void setChildren(List<FileTree> children) {
		this.children = children;
	}
	public String getFolderpath() {
		return folderpath;
	}
	public void setFolderpath(String folderpath) {
		this.folderpath = folderpath;
	}
}

File: src/main/java/com/github/izerui/file/vo/FileTree.kt
package com.github.izerui.file.vo

import java.io.Serializable

class FileTree : Serializable {
    var foldername: String? = null//文件夹名字
    var folderpath: String? = null//文件夹实际路径
    var children: List<FileTree>? = null//子文件夹
}
